src/NLog/Targets/FileTarget.cs

```diff
@@ -486,6 +486,11 @@ public bool ArchiveOldFileOnStartup
             set => _archiveOldFileOnStartup = value;
         }
         private bool? _archiveOldFileOnStartup;
+        
+        /// <summary>
+        /// Gets or sets a value indicating whether the header should be written on every startup.
+        /// </summary>
+        public bool WriteHeaderOnStartup { get; set; }
```

Comment:
Please rename to `WriteHeaderOnInitialFileOpen` so name matches the behavior.

Please change XML-docs to:
```
Gets or sets whether to write <see cref="Header"/> on initial creation of file appender. Independent of file is empty or not.
Alternative use <see cref="ArchiveOldFileOnStartup"/> to ensure each application session receives its own individual log-file.
```

Reply:
I don't know why, but i can't add reference to "Header" in XML-doc